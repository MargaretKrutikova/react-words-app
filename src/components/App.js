import React from 'react';
import * as api from '../api';

class App extends React.Component {
  render() {
    return (
      <div className="App">
        <h1>React Template Project</h1>
        <p>Let's get started!</p>
      </div>
    );
  }
}

export default App;