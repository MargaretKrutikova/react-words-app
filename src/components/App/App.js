import React from 'react';
import WordsContainer from '../Words/WordsContainer';

class App extends React.Component {
  render() {
    return (
      <div className="App container">
        <h1 className="col-xs-12">Word list</h1>

        <div className="col-xs-12 col-sm-12 col-md-6" >
          <WordsContainer />
        </div>
      </div> 
    );
  }
}

export default App;