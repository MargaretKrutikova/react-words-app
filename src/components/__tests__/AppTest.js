import 'babel-polyfill'; // required for Object.values

import React from 'react';
import Enzyme from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

import App from '../App';

Enzyme.configure({ adapter: new Adapter() });

describe('App', () => {
  it('renders correctly', () => {
    const wrapper = Enzyme.shallow(<App />);

    expect(wrapper.children().length).toBe(2);
    // snapshot expectations
    expect(wrapper).toMatchSnapshot(); // if anything changes the snapshot expectation will fail
  });
});