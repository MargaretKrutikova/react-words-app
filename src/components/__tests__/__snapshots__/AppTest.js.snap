// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <App />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <h1>
          React Template Project
        </h1>,
        <p>
          Let's get started!
        </p>,
      ],
      "className": "App",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "React Template Project",
        },
        "ref": null,
        "rendered": "React Template Project",
        "type": "h1",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Let's get started!",
        },
        "ref": null,
        "rendered": "Let's get started!",
        "type": "p",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h1>
            React Template Project
          </h1>,
          <p>
            Let's get started!
          </p>,
        ],
        "className": "App",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "React Template Project",
          },
          "ref": null,
          "rendered": "React Template Project",
          "type": "h1",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Let's get started!",
          },
          "ref": null,
          "rendered": "Let's get started!",
          "type": "p",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
